### ISSUE TITLE
about code data

### ISSUE THREAD

**Issue by chrislouis0106** [THU-KEG/MetaKGR#4]
About NELL dataset, could you please give an introduction about the meaning of files. So I want to know how to deal with the original data about FB15K-237.
Other aspects, about NELL dataset, I find some details in the paper " 2018 - DeepPath A Reinforcement Learning Method for Know reason". But it's different from your dataset. So I'm curious to how to get the dataset. Or how to download the original dataset of NELL.
Thank you a lot!

**Comment by davidlvxin** [comment]
I did some processing on the NELL data set because the test set of the original NELL-995 data set only contains a small number of relations. Specifically, I randomly split the ``train.txt`` of NELL-995 to generate new training set, validation set and test set. If you need the original NELL-995 data set, you can download it [here](https://github.com/salesforce/MultiHopKG).

**Comment by chrislouis0106** [comment]
thank you very much. your work is very excellent.&nbsp;



---Original---
